templates/base.html

```diff
@@ -2,32 +2,35 @@
     <head>
         <title>Wikipedia Speedruns!</title>
 
-        <script src="{{url_for('static', filename='js/base.js')}}" defer></script>
-        <link rel= "stylesheet" type= "text/css" href="{{url_for('static', filename='stylesheets/main.css')}}">
-
-        <!--Bootstreap stuff-->
+        <!--Bootstrap stuff-->
         <meta charset="utf-8">
         <meta name="viewport" content="width=device-width, initial-scale=1">
         <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
         <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.7.2/font/bootstrap-icons.css">
         <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js" integrity="sha384-ka7Sk0Gln4gmtz2MlQnikT1wXgYsOg+OMhuP+IlRH9sENBO0LRn5q+8nbTov4+1p" crossorigin="anonymous"></script>
 
-        {% if username %}
-        <script defer>
-            const user_id='{{user_id}}';
-        </script>
-        {% else %}
-        <script defer>
-            const user_id = null;
+        <script type="module">
+            import {setServerData} from "/static/js/modules/serverData.js";
```

Comment:
Might need to be `./static/.../` ? Not sure

Reply:
no, because we need an absolute path, the relative one changes depending on which page we visited,